Assets/Project/Scripts/Utils/Definitions/EFailureReasonType.cs

```diff
@@ -0,0 +1,11 @@
+namespace Project.Scripts.Utils.Definitions
+{
+    public enum EFailureReasonType
+    {
+        Tornado,
+        Meteorite,
+        AimingMeteorite,
+        Thunder,
+        Others
```

Comment:
```suggestion
        Others = -1
```
Gimmick増えた時にindex変わるのも微妙な感じがするので、こうしたい。

Reply:
お，Others = -1 にすると，index をどう固定できるの？普通は上から 0, 1, 2, ... と振られていくよね？

Reply:
ギミック追加するときに `Others` の前から追加すれば問題ない

Reply:
列挙型の根幹が index 管理だということだよね．まぁ，Others を先頭に持ってくればミスもないし良いんじゃないかな？

Reply:
良いと思う

Reply:
良いと思います